ipdb/__main__.py

```diff
@@ -47,7 +47,9 @@
 debugger_cls = shell.debugger_cls
 
 
```

Comment:
get_context_from_config call below will default to 3 if a set context cannot be found